### ISSUE TITLE
Start IPR error: prman_for_python Not defined

### ISSUE THREAD

**Issue by userx-bw** [prman-pixar/RenderManForBlender#579]
![screenshot_renderman__2018-08-07_08-59-29](https://user-images.githubusercontent.com/6270942/43780831-20817d7c-9a21-11e8-8c8e-b53582f28e73.png)
I am running Renderman 20.7 the nodes seem to work but when clicking start IPR to check it, I get that error. Click render it gets bank screen. not even a cube.

**Comment by adminradio** [comment]
Reinstall the addon without any space or hyphen in the addons directory name (in this case, remove "-master" from directory name. It should run now.

**Comment by userx-bw** [comment]
I rezipped the directory, it caused an error, so I went and installed the otginal again, changed the directory by removing the  -master part, closed and restarted Blender, it seen it, loaded the add-on then I got this again. 
![addon_notworking](https://user-images.githubusercontent.com/6270942/43847906-61b66aa6-9af7-11e8-9875-0455cc95a1c8.png)
This is for version 20.7 and the addon description states 20.5 still. so maybe a update is needed, i do not know.
